test/topology_custom/test_tablets.py

```diff
@@ -301,3 +301,51 @@ async def test_read_of_pending_replica_during_migration(manager: ManagerClient,
 
     rows = await cql.run_async("SELECT pk from test.test")
     assert len(list(rows)) == 1
+
+
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+async def test_alter_dropped_tablets_keyspace(manager: ManagerClient) -> None:
+    config = {
+        'enable_tablets': 'true'
+    }
+
+    logger.info("starting a node (the leader)")
+    servers = [await manager.server_add(config=config)]
+
+    logger.info("starting a second node (the follower)")
+    servers += [await manager.server_add(config=config)]
+
+    await manager.get_cql().run_async("create keyspace ks with "
+                                      "replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 1} and "
+                                      "tablets = {'enabled': true}")
+    await manager.get_cql().run_async("create table ks.t (pk int primary key)")
+
+    logger.info(f"injecting wait-before-handling-keyspace-rf-change into the leader node {servers[0]}")
+    injection_handler = await inject_error_one_shot(manager.api, servers[0].ip_addr,
+                                                    'wait-before-handling-keyspace-rf-change')
+
+    async def alter_tablets_ks_without_waiting_to_complete():
+        res = await manager.get_cql().run_async("select data_center from system.local")
+        this_dc = res[0].data_center  # ALTER tablets KS needs to have the DC specified explicitly
```

Comment:
Why do you need to get DC from CQL though? Note that the node you get DC from, and the one you execute the statement later, could be different, so if they booted in different DCs, this code would not make much sense. Well it's not a problem in this test because they should boot in the same DC. I guess the only thing you want to do here is to ensure you pass a DC that exists? (and don't care which DC in particular it is) You could make it more obvious in the comments.

Reply:
I need to explicitly state the name of the DC, because ALTER tablets KS doesn't accept the generic `replication_factor` tag.

Reply:
Changed the comment to be more specific: `# ALTER tablets KS only accepts a specific DC, it rejects the generic 'replication_factor' tag`